test/model/src/driver/debug_log_parser.cpp

```diff
@@ -229,14 +258,75 @@ debug_log_parser::metadata_apply(const row_put &op)
 
     /* Special handling for the system prefix. */
     else if (starts_with(key, "system:")) {
-        /* We don't currently need to handle this. */
+
+        /* Set the base write generation. */
+        if (key == "system:checkpoint_base_write_gen") {
+            _base_write_gen = m->get_uint64("base_write_gen");
+        }
+
+        /* Handle checkpoints. */
+        else if (starts_with(key, "system:checkpoint") || starts_with(key, "system:oldest")) {
+            size_t p = key.find('.');
+            std::string ckpt_name = p == std::string::npos ? WT_CHECKPOINT : key.substr(p + 1);
```

Comment:
I'm not familiar with the naming, does the key for a named checkpoint look like "system:checkpoint.name" or "system:oldest.name"?  Perhaps a comment?  Also, it doesn't look like there's any difference in this code between "system:checkpoint" and "system:oldest".  Is that expected?